mcs/tools/upload-to-sentry/UploadToSentry.cs

```diff
@@ -411,12 +414,17 @@ public static void Main (string[] args)
 
 			var dsn = new Dsn(url);
 
+			var files = GetFiles (fileRoot);
+
+			if (files.Count () == 0)
```

Comment:
This could be expensive operation, try to avoid it

Reply:
How? `Directory.GetFiles()` already returns a fully materialized list so I don't see how this could expensive.

Reply:
Sadly, it is not. Array does not implement `ICollection`/`ICollection<T>`, so it does not get fast paths.

Reply:
I can switch to `.Empty()` if it makes you feel better :)

Reply:
Please change the GetFiles to return the string array and use Length here instead